spec/examples.txt

```diff
@@ -0,0 +1,92 @@
+example_id                                                 | status  | run_time        |
```

Comment:
Should this file be committed? 🤔 

It looks like an artifact of running specs locally.